Lib/test/test_site.py

```diff
@@ -585,6 +591,15 @@ def test_startup_interactivehook_isolated_explicit(self):
             'import site, sys; site.enablerlcompleter(); sys.exit(hasattr(sys, "__interactivehook__"))']).wait()
         self.assertTrue(r, "'__interactivehook__' not added by enablerlcompleter()")
 
+    @classmethod
+    def _calc_sys_path_for_underpth_nosite(self, sys_prefix, lines):
+        sys_path = []
+        for line in lines:
+            if not line or line[0] == '#':
+                continue
+            abs_path = os.path.abspath(os.path.join(sys_prefix, line))
+            sys_path.append(abs_path)
+        return sys_path
```

Comment:
This seems unused?

Reply:
You're right removed it.
I couldn't find where to put the last parts of it: https://bugs.python.org/file46433/underpth_blank_lines.diff